views/tables.ejs

```diff
@@ -56,7 +62,7 @@
           <a href='/tables/<%= data[i].TableName %>'>
             <%= data[i].TableName %>
           </a>
-          <a href="/" onClick="deleteTable('<%= data[i].TableName %>')"
+          <a href="#" onClick="deleteTable('<%= data[i].TableName %>')"
```

Comment:
I would still return false as otherwise browser will navigate to # and while it won't affect page, it will affect history navigation.

Reply:
href="#" normally does not trigger page navigation. It is an internal page reference so the page does not reload.
https://www.w3.org/Style/Examples/007/target.en.html

Should I keep the "return false" to stay on the safe side ?

Reply:
It doesn't navigate to a different page but it does add the hash symbol to the URL and adds history entry in the browser if event is not prevented.